apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -204,6 +206,28 @@ export const EventSetupTab = (
             )}
           </ul>
         )}
+        {validLocations.some((location) => location.type === "integrations:google:meet") && (
+          <div className="flex">
+            <Icon.FiAlertTriangle className="mr-4" />
+            <p className="text-sm">
+              In order to use Google Meet you must set your{" "}
+              <a
+                href="javascript:;"
```

Comment:
nit: A button is more appropriate for a javascript action. https://blogs.ancestry.com/techroots/buttons-vs-anchors/. It is not good for a11y also I think

Reply:
Not sure what happened here but I replaced the `<a>` tag with `<Link>` thanks to @sean-brydon feedback.